packages/grafana-ui/src/components/Drawer/Drawer.tsx

```diff
@@ -66,31 +70,30 @@ export function Drawer({
     setIsOpen(true);
   }, []);
 
-  const content = <div className={drawerStyles.content}>{children}</div>;
-  const style: CSSProperties = {};
-  if (inline) {
-    style.position = 'absolute';
-  }
+  // deprecated width width prop now defaults to empty string which make the size prop take over
+  const fixedWidth = isExpanded ? '100%' : width ?? '';
+  const useSizeWidth = !fixedWidth && !isExpanded;
```

Comment:
The only way for `fixedWidth` to be `falsy` is if `isExpanded` is also `false`, so I think `useSizeWidth = !fixedWidth` is enough no?

Reply:
good catch :) fixed!